package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.2.15",
+  "version": "5.3.0",
```

Comment:
I'm not sure why you increased the minor release number instead of the patch number.

Anyways, it's fine this time.

Reply:
i told her to bump it up 5.3.0 because i assumed all of the WoN was incrementing from 5.2.1 all the way up to .14

i wanted to start anew with 5.3.0    let us know if you want the next version to go back to 5.2.15

Reply:
So 5.3.0 is "post WoN", lol.

Just leave it. It's fine.

Reply:
Actually, your decision gives us the option to continue hotfixes on 5.2.x.